collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/DeleteBlankFormFragment.kt

```diff
@@ -80,11 +85,17 @@ class DeleteBlankFormFragment(
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        appBarLayout = requireActivity().findViewById(org.odk.collect.androidshared.R.id.appBarLayout)
         val blankFormListMenuProvider =
             BlankFormListMenuProvider(requireActivity(), blankFormListViewModel)
         menuHost.addMenuProvider(blankFormListMenuProvider, viewLifecycleOwner, State.RESUMED)
     }
 
+    override fun onResume() {
+        super.onResume()
+        appBarLayout?.setLiftOnScrollTargetView(list)
```

Comment:
Why does this call end up in `onResume` and not `onViewCreated`?

Reply:
`onViewCreated` is called once but we need to update the view that appbar observes every time we switch between blank and saved forms (it is a viewpager) because we have two lists with the same id.